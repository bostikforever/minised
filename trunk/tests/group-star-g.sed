s/\(ab\)*/_\1_/g
