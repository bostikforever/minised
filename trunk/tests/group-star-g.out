__x_ab_y_ab_z__
__c__
